lib/openfoodfacts.dart

```diff
@@ -147,6 +148,48 @@ class OpenFoodAPIClient {
     return Status.fromApiResponse(response.body);
   }
 
+  /// Temporary: saves product packagings V3 style.
+  ///
+  /// For the moment that's the only field supported in WRITE by API V3.
+  /// Long term target is of course more something like [saveProduct].
+  static Future<ProductResultV3> temporarySaveProductV3(
```

Comment:
```suggestion
  static Future<ProductResultV3> temporarySaveProductPackagingsV3( 
```
Just a suggestion

Reply:
Could make sense but in the long run I expect more fields to match V3 (e.g. one by month) : I'm not going to create a different method for each.